build_debian.sh

```diff
@@ -746,7 +746,16 @@ if [[ $TARGET_BOOTLOADER == uboot ]]; then
     elif [[ $CONFIGURED_ARCH == arm64 ]]; then
         if [[ $CONFIGURED_PLATFORM == pensando ]]; then
             ## copy device tree file into boot (XXX: need to compile dtb from dts)
-            sudo cp -v $PLATFORM_DIR/pensando/elba-asic-psci.dtb $FILESYSTEM_ROOT/boot/
+            sudo cp -v $FILESYSTEM_ROOT/usr/lib/linux-image-${LINUX_KERNEL_VERSION}-${CONFIGURED_ARCH}/pensando/elba-asic-psci.dtb $FILESYSTEM_ROOT/boot/
+            sudo cp -v $FILESYSTEM_ROOT/usr/lib/linux-image-${LINUX_KERNEL_VERSION}-${CONFIGURED_ARCH}/pensando/elba-asic-psci-lipari.dtb $FILESYSTEM_ROOT/boot/
+            sudo cp -v $FILESYSTEM_ROOT/usr/lib/linux-image-${LINUX_KERNEL_VERSION}-${CONFIGURED_ARCH}/pensando/elba-asic-psci-mtfuji.dtb $FILESYSTEM_ROOT/boot/
+            sudo cp -v $PLATFORM_DIR/pensando/install_file $FILESYSTEM_ROOT/boot/
+            if [ -e $PLATFORM_DIR/pensando/uboota.img ]; then
+                sudo cp -v $PLATFORM_DIR/pensando/uboota.img $FILESYSTEM_ROOT/boot/
```

Comment:
i do not under why sonic image needs uboota.img and boot0.img. who build these?

Reply:
These are not built as a part of SONiC image. These are provided by us in case the vendor wants to upgrade their uboot and boot0 image when upgrading to a new SONiC image. Its not mandatory to provide these.

Reply:
@shanshri can we remove the Uboot from sonic image since that is not needed?

Reply:
We would like to like keep this as an option to the vendor. If they want to upgrade uboot with SONiC installation(which would be needed in some use cases) they can just copy the uboot image in this directory before generating a SONiC image.

Reply:
Removed the uboot support for now as discussed. Will raise a separate PR for it.